apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/rivers/DistribuerBrevRiver.kt

```diff
@@ -17,7 +17,7 @@ internal class DistribuerBrevRiver(
     private val logger = LoggerFactory.getLogger(DistribuerBrevRiver::class.java)
 
     init {
-        initialiserRiver(rapidsConnection, BrevEventTypes.JOURNALFOERT.toEventname()) {
+        initialiserRiver(rapidsConnection, BrevHendelseHendelseType.JOURNALFOERT.lagEventnameForType()) {
```

Comment:
Forslag (men kanskje til neste PR for å ikkje ta alt på ein gong): Endre `initialiserRiver`-metoden til å ta inn ein `EventnameHendelseType` heller enn string som parameter, og så flytte `lagEventnameForType`()-kallet inn dit

Reply:
Lurt! Skriver det opp i blokka, har en del andre endringer ventende og så tar det i egen PR!